resources/local/config.toml

```diff
@@ -6,6 +6,8 @@
 # If set, use this hash as a trust anchor when joining an existing network.
 #trusted_hash = 'HEX-FORMATTED BLOCK HASH'
 
+archival_sync = false
```

Comment:
This should be `true` as well (as per https://github.com/casper-network/casper-node/pull/1995/files#r697618855).

Reply:
Still, the archival sync should be enabled (`true`) by default.

Reply:
This should be set to `true`.